compiler/src/iree/compiler/Dialect/Flow/Transforms/FormDispatchRegions.cpp

```diff
@@ -375,12 +405,12 @@ static bool hasCompatibleOuterParallelLoops(
   }
 
   /// Project out the non-parallel dimensions.
-  llvm::SmallBitVector producerProjectedDims(producerParallelLoops);
+  llvm::SmallBitVector producerProjectedDims(rootOuterParallelLoops);
```

Comment:
In the case of fusing with the producer, `rootOuterParallelLoops` will be the consumer parallel loop? Is it fine to always use `rootOuterParallelLoops` here?

Reply:
Yes, we are checking here that just taking the outer parallel loops of the root op (we have checked that the outer parallel loops of producer and consumer match the root op) if the indexing map is indentity on both .Thats sufficient condition to know that both the producer and the consumer have compatible outer parallel loops with the root op (transitively).